src/NuGet.Status/.editorconfig

```diff
@@ -0,0 +1,12 @@
+[*.cs]
```

Comment:
Should this be at the root of the repo like the other repos?

Reply:
If it's in the root, then it will apply to future projects added, right? That seems more future-proof to me.